src/common/target.c

```diff
@@ -168,7 +168,6 @@ static const TargetEntry TargetMap[] = {
     {   "sim6502",      TGT_SIM6502     },
     {   "sim65c02",     TGT_SIM65C02    },
     {   "supervision",  TGT_SUPERVISION },
-    {   "vc20",         TGT_VIC20       },
```

Comment:
If you are using undocumented features you are on your own. C'est la vie.
We already have many targets, we will get even more and I want clean source.
At least I mentioned the VC20 in vic20.sgml.